src/plugins/oSnap/components/TransactionBuilder/TransferFunds.vue

```diff
@@ -23,10 +23,9 @@ const emit = defineEmits<{
   updateTransaction: [transaction: TransferFundsTransaction];
 }>();
 
-const nativeAsset = getNativeAsset(props.network);
 const amount = ref(props.transaction.amount ?? '');
 const recipient = ref(props.transaction.recipient ?? '');
-const tokens = ref<Token[]>([nativeAsset, ...props.tokens]);
+const tokens = ref<Token[]>(props.tokens);
```

Comment:
i think we need to keep the native asset as first token in the list. is that possible?